internal/host/host.go

```diff
@@ -312,6 +313,24 @@ func (m *Manager) UpdateInventory(ctx context.Context, h *models.Host, inventory
 	}).Error
 }
 
+func (m *Manager) checkValidationChanged(ctx context.Context, h *models.Host, newValidationRes validationsStatus) (validationsStatus, bool, error) {
+	var currentValidationRes validationsStatus
+	if h.ValidationsInfo != "" {
```

Comment:
what will happen if h.ValidationsInfo == "" && newValidationRes != "" ? we should also update the DB in this case

Reply:
The case you describe means that currentValidationRes != newValidationsRes therefore it will be handled in line 323